packages/valory/services/optimus/service.yaml

```diff
@@ -128,3 +132,9 @@ cert_requests:
   not_before: '2022-01-01'
   public_key: ${ACN_NODE_PUBLIC_KEY:str:02d3a830c9d6ea1ae91936951430dee11f4662f33118b02190693be835359a9d77}
   save_path: .certs/acn_cosmos_11000.txt
+---
+public_id: valory/http_server:0.22.0
+type: connection
+config:
+  host: ${HTTP_SERVER_HOST:str:0.0.0.0}
+  port: ${HTTP_SERVER_PORT:int:8716}
```

Comment:
@DIvyaNautiyal07 @0xArdi Let me know if this is OK for the "normal" Optimus.

Reply:
@jmoreira-valory seems fine to me